frontend-client/src/components/BodyRefboxLog.vue

```diff
@@ -1,20 +1,92 @@
 <template>
-  <div class="container-fluid border p-0 refbox-log overflow-auto text-left">
-    <div class="m-2">
-      
+  <div class="container-fluid  border p-0 refbox-log overflow-auto text-left">
+    <div class="d-flex justify-content-end">
+      <button @click=connect class="btn btn-primary "> Connect </button>
+      <button @click=disconnect class="btn btn-primary  ">
+        DC
+      </button>
+      <button @click=send class="btn btn-primary ">Send Msg</button>
+    </div>
+    
+    <div class="mx-3 mt-3">
+      <div v-for="(msg,index) in msgArray" :key=index>
+          <h6 v-if="msg.level !== 'attention' " class= "mb-0" :class="setClassName(msg.level)">{{msg.time}} [{{msg.component}}]: {{msg.message}}</h6>
```

Comment:
I would suggest using the same font size for these messages and attention messages and to use a monospaced font

Reply:
Thanks for the suggestion. I did make all the logged messages the same size. As for the font, it is already a monospaced font (Helvetica)

Reply:
Thank you! I think the same size makes it look more unified and doesn't break the flow as much.

Helvetica is not a monospaced font but a sans-serif, variable-width font. It is also not available on all devices (especially Linux distributions, because Helvetica licenses are not free, these usually default to a generic sans-serif font). Monospaced fonts include Menlo, Courier New, Roboto Mono, Source Code Pro, etc.